contract/services.py

```diff
@@ -415,6 +415,39 @@ def delete(self, contract):
         except Exception as exc:
             return _output_exception(model_name="Contract", method="delete", exception=exc)
 
+    @check_authentication
+    def terminate_contract(self, contract):
+        try:
+            # TODO - add this service to the tasks.py in apscheduler once a day
+            #  to check if contract might be terminated
+            contract_to_terminate = ContractModel.objects.filter(id=contract["id"]).first()
```

Comment:
Why not having a queryset such as 
ContractModel.objects.filter(date_valid_to__lt=datetime.date.now()).filter(state=7) --> this might happen for many contracts

Reply:
Ok! I will change! It is a good idea when we want terminate not only one contract. Good idea!